k8s/hyperconverged/tutorial-configure-openebs-gke.md

```diff
@@ -1,76 +1,132 @@
-```
-sudo cat /etc/iscsi/initiatorname.iscsi
-sudo service open-iscsi status
-```
+ OpenEBS uses iSCSI to connect to the block volumes. Select one of the nodes in the cluster and click **SSH**, to open the node in a terminal.
+
+1. Check if the initiator name is configured.
+
+  ```bash
+
+    ~$ sudo cat /etc/iscsi/initiatorname.iscsi
+    ## DO NOT EDIT OR REMOVE THIS FILE!
+    ## If you remove this file, the iSCSI daemon will not start.
+    ## If you change the InitiatorName, existing access control lists
+    ## may reject this initiator.  The InitiatorName must be unique
+    ## for each iSCSI initiator.  Do NOT duplicate iSCSI InitiatorNames.
+    InitiatorName=iqn.1993-08.org.debian:01:6277ea61267f
+
+  ```
+
+2. Check if the iSCSI service is running.
+
+  ```bash
+
+   ~$ sudo service open-iscsi status
+   ● open-iscsi.service - Login to default iSCSI targets
+     Loaded: loaded (/lib/systemd/system/open-iscsi.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Tue 2017-10-24 14:33:57 UTC; 3min 6s ago
+       Docs: man:iscsiadm(8)
+             man:iscsid(8)
+    Main PID: 1644 (code=exited, status=0/SUCCESS)
+       Tasks: 0
+      Memory: 0B
+         CPU: 0
+      CGroup: /system.slice/open-iscsi.service
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 systemd[1]: Starting Login to default iSCSI targets...
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 iscsiadm[1640]: iscsiadm: No records found
+   Oct 24 14:33:57 gke-cluster-3-default-pool-8b0f2a27-5nr2 systemd[1]: Started Login to default iSCSI targets.
+
+  ```
+
+3. Repeat steps 1 and 2 for the remaining nodes.
 
 ## Step 2 : Run OpenEBS Operator (via Google Cloud Shell)
 
 Download the latest OpenEBS Operator Files
-```
+
+```bash
 git clone https://github.com/openebs/openebs.git
 cd openebs/k8s
 ```
 
-Setup the kubectl to run in admin context. [See Appendix below for creating an admin context in GCP](https://github.com/openebs/openebs/blob/master/k8s/hyperconverged/tutorial-configure-openebs-gke.md#setting-kubernetes-cluster-admin-context). The below command will prompt for username/password. Provide username as "admin" and password for the admin can be obtained from : Google Cloud Platform -> Container Engine -> (cluster) -> Show Credentials
+### Setting kubernetes cluster admin context
+
+To create/modify service accounts and grant previleges, kubectl needs to be run with admin previleges. The following commands help to setup and use the admin context for Google Container Engine via the Google Cloud Shell.
+
+#### Step 1: Initialize credentails to allow kubectl to execute commands on the container cluster
+
+```bash
+gcloud container clusters list
+gcloud container clusters get-credentials demo-openebs03 --zone us-central1-a
+```
+
+#### Step 2 : Setup the admin context
+
+- Access the credentails information from : Google Cloud Platform -> Container Engine -> (cluster) -> Show Credentials. 
```

Comment:
remove the lines:
- access the credentials
- save the cluster CA

These are not required, since we are using username password